File: 프로그래머스/unrated/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.py
def solution(my_str, n):
    answer = []
    string=""
    for i in range(len(my_str)):
        if i % n == 0 and i != 0:
            answer.append(string)
            string=""
        string += my_str[i]
    answer.append(string)
    return answer

File: 프로그래머스/0/120913. 잘라서 배열로 저장하기/잘라서 배열로 저장하기.java
class Solution {
    public String[] solution(String my_str, int n) {
        String[] answer;
        if(my_str.length() % n == 0)
            answer = new String[my_str.length()/n];
        else
            answer = new String[my_str.length()/n + 1];
        
        StringBuffer sb = new StringBuffer();
        for(int i=0;i<my_str.length();i++){
            if(i%n == 0 && i != 0){
                answer[i/n-1] = sb.toString();
                sb.delete(0,n);
            }
            sb.append(my_str.charAt(i));
        }
        
        if(my_str.length() % n != 0)
            answer[my_str.length()/n] = sb.toString();
        else
            answer[my_str.length()/n-1] = sb.toString();
        
        return answer;
    }
}